bosh-cf-crossregion/manifests/cross.yml

```diff
@@ -0,0 +1,1391 @@
+# All certs which are used in this file are for testing.
+# You should use your certs to replace them for production.
+---
+name: cross-region
+director_uuid: REPLACE_WITH_DIRECTOR_ID
+
+releases:
+- {name: cf, version: 231}
+
+networks:
+- name: cf_private
+  type: manual
+  subnets:
+  - range: 10.0.16.0/20
+    gateway: 10.0.16.1
+    dns: [168.63.129.16]
+    reserved: ["10.0.16.2 - 10.0.16.3"]
+    static: ["10.0.16.4 - 10.0.16.200"]
+    cloud_properties:
+      virtual_network_name: primaryboshvnet-crp
+      subnet_name: CloudFoundry
```

Comment:
You don't need to put `subnetNameForCloudFoundry` into `parameters`. It's OK to put it in `variables`.

But if it's hardcoded in `cross.yml` and users change `subnetNameForCloudFoundry` in `variables`, the manifest won't work. So, it's better to change to `REPLACE_WITH`. Then the script can update it to the value of `subnetNameForCloudFoundry` in `variables`.
Does it make sense?